### DISCUSSION TITLE
Linux Pinta is far more stable

### DISCUSSION THREAD

**Discussion by AshleyPikachu** [PintaProject/Pinta#943] [Q&A] [answered]
Hi, just wanted to say I'm on Rocky 9 now, and yeah so far so good when using the Pinta flatpak. It has been far more stable on Linux than it has been on Windows or Mac. I have some questions below and thank you for all the hard work.

Questions
1. When zooming in with Ctrl+scroll is it possible for the camera to not move to the top left?
2. Can I merge and add layers from the list of layers itself by right clicking?
3. Can I use Bicubic scaling?
3. Is it possible to resize a paste image by pulling and stretching them instead of using the Resize Image button.
4. Can I add a tint to areas selected in the magic wand tool to see them better?

**Comment by cameronwhite** [comment]
1. This is  tracked in bug #681
2. There isn't any right click option currently - you'd have to use the buttons at the bottom of the layers list
3. The resize dialog supports either bilinear or nearest neighbor filtering, these options come from the Cairo library (https://cairographics.org/manual/cairo-cairo-pattern-t.html#cairo-filter-t)
4. I think this sounds similar to  bug #585 for improving how resizing works. Do you mean resizing just the selected pixels after pasting, or adjusting the actual image / canvas size?
5. There isn't a way to do this, but would be a good feature request to submit

**Reply by AshleyPikachu** [reply, marked as answer]
1. Alright, thanks for the heads up.
2. No worries thanks for letting me know
3. Cool none the less
4. Yeah I mean resizing just the selected pixels.
5. Alright I'll go submit that thanks!

Thanks for all the help :)
